_posts/2015-06-23-hardly-anyone-knows-cd.markdown

```diff
@@ -0,0 +1,40 @@
+
+The conversations in a trade show booth generally start with the visitor asking what we do (gotta work more on that so they don’t have to) and me telling them that Go is a continuous delivery server. From there we go on to talk about what makes Go unique and why they should use it.
+
+At this show, when I told people Go is a continuous delivery server I was met with mostly blank stares. This was a conference attended by 100% people who create software for a living. The people attending care enough about their craft to spend (or get their company to spend) a couple thousand US dollars to come. But they had no idea what Continuous Delivery really is. I probably should note, this isn't meant as a knock on that conference at all.
+
+The lack of knowledge is a really bad thing. Not just for the Go CD project, but for software in general. The world runs on software. Too much of that software is bad. The practices around Continuous Delivery could make some of it better, or kill it before it gets out.
+
+####So what can we do?
+
+Buy or borrow a copy of [Continuous Delivery](http://www.amazon.com/Continuous-Delivery-Deployment-Automation-Addison-Wesley/dp/0321601912) by Jez Humble and Dave Farley for your office. Make everyone read at least the chapters that apply to them. Yes, Jez and Dave both worked for ThoughtWorks when they were writing the book. Yes, Jez was the product owner of Go before the book came out. No, we won't make any money off the link if you buy it. I promise this isn’t bias, it’s the definitive work on the subject.
+
+Get [The Phoenix Project](http://www.amazon.com/The-Phoenix-Project-Helping-Business/dp/0988262509/ref=pd_bxgy_14_img_y) by Gene Kim. It’s a fictional novel and a bit corny at times, but people will learn a bit even if they don’t mean to.
+
+Send people that don’t know about Continuous Delivery to conferences that are specific to CD and DevOps. My favorite is [DevOpsDays](http://www.devopsdays.org/). You don't need huge, expensive conferences where you’ll have to get finance approval to attend. The next one I’m going to is 200 bucks. If they don’t have one in your area create one or find someone that will. (FYI, if anyone in Seattle is interested in doing that let me know)
+
+Take a friend who’s CD impaired to a [DevOps Meetup](http://devops.meetup.com/). As I’m writing this there are groups in 404 cities worldwide at that link alone. Trying to get your meetup going and struggling for content and/or speakers? Tell me, I know a few people and might be able to help.
+
+Stop assuming everyone knows what we’re talking about when we talk about CD. Many of them are smiling and nodding the same way I do when my mother talks about her flowers.
+
+Feel free to comment with your own resource, this isn't even close to a definite list.
+
+####One last thing...
+
+Stop telling people that the phrases DevOps and Continuos Delivery are overused. They aren’t. Hardly anyone knows what Continuous Delivery is.
```

Comment:
Rephrase as "Not everyone knows what CD is". I'm just trying to mellow it down a bit :)